apps/mobile/src/features/approver/components/fees/base-fee-card.tsx

```diff
@@ -1,12 +1,9 @@
 import { ReactElement } from 'react';
 
 import { Balance } from '@/components/balance/balance';
-import { useDynamicFeeFlag } from '@/features/feature-flags';
```

Comment:
I don't know if we will ever implement this now so you can remove `useDynamicFeeFlag` altogether?

Reply:
I'll remove it. We'll add it back if anything.